src/components/breadcrumbs/_breadcrumbs.scss

```diff
@@ -90,6 +90,10 @@
         height: auto;
         border: 0;
         color: $chevron-border-colour;
+
+        // IE8 doesn't seem to like rendering pseudo-elements using @font-faces,
+        // so fall back to using any sans-serif font to render the chevron.
+        font-family: sans-serif;
```

Comment:
Is this safe? Browser do weird things, I wonder if picking something stable like helvetica/arial would be good to start with.

Reply:
that would be forcing either a default instead of browser/os default combo

That said, our font stack definition is `$govuk-nta-light: "nta", Arial, sans-serif;`, so by going by that we either do `sans-serif` or `Arial, sans-serif`

Not sure how up to date this table is https://www.granneman.com/webdev/coding/css/fonts-and-formatting/web-browser-font-defaults/

Reply:
Yeah, was thinking we use the current font stack minus nta

Reply:
Updated to use `Arial, sans-serif`.